lib/media/drm_engine.js

```diff
@@ -1216,28 +1210,6 @@ shaka.media.DrmEngine.prototype.unpackPlayReadyRequest_ = function(request) {
 };
 
 
-/**
- * Format FairPlay license requests.  Modifies the request object.
- * @param {shakaExtern.Request} request
- * @param {string} contentId
- * @private
- */
-shaka.media.DrmEngine.prototype.formatFairPlayRequest_ =
```

Comment:
The demo in the FPS SDK does this, but I believe it is application-specific business logic. None of the Fairplay license proxies I work with expect this behaviour from clients.